executor/runtime/docker/docker.go

```diff
@@ -1144,14 +1148,13 @@ func (r *DockerRuntime) Prepare(ctx context.Context, pod *v1.Pod) error { // nol
 			// probably because if failed to be pulled, potentically indicating that the image doesn't exist,
 			// or a bad deploy, or heck just a typo in the name or something
 			if sidecarConfig.Required {
-				err = fmt.Errorf("Unable to get volume container of required sidecar %s", sidecarConfig.ServiceName)
-				goto error
-			} else {
-				// If the ContainerName is still uninitialized, but this service is not required, then it is
-				// ok to just log about it, but it should not be included on the list of volume containers to use
-				// (because it doesn't exist)
-				logger.G(ctx).Warnf("Skipping volume container of optional sidecar %s", sidecarConfig.ServiceName)
+				return fmt.Errorf("Unable to get volume container of required sidecar %s", sidecarConfig.ServiceName)
```

Comment:
This is an example of something that won't work without the named return value.

Reply:
Oh right. Yuck.